src/components/shared/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,25 @@
+import CrossThick from "@/icons/CrossThickIcon";
+import { Announcement } from "@/models/Announcement";
+
+interface props {
+  announcement: Announcement;
+}
+
+export default function AnnouncementBanner({ announcement }: props) {
+  return (
+    <div
+      className="flex w-[281px] md:w-[361px] p-4 items-start gap-4 rounded-sm bg-neutral-white border border-other-stroke"
+      style={{ boxShadow: "0px 10px 10px 0px rgba(36, 41, 46, 0.15)" }}
```

Comment:
The design system contains shadows. Can you add those to the theme and use them instead? We should avoid using raw CSS.